server/model/survey_response.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	mmModel "github.com/mattermost/mattermost/server/public/model"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-user-survey/server/utils"
+)
+
+const (
+	ResponseTypeComplete = "complete"
+	ResponseTypePartial  = "partial"
+)
+
+type SurveyResponse struct {
+	ID           string            `json:"ID"`
```

Comment:
Ah, probably typo or auto-generated code. Fixed to lowercase `id`.

Reply:
ah, about camel case, yeah we can do anything. With camel case, since the webapp's types are also in camel case, it nicely matches it. I checked MS Teams plugin as well and they are also using camel case, so we're fine as it doesn't seem to be an established convention.